src/temgymbasic/components.py

```diff
@@ -635,8 +719,9 @@ class Biprism(Component):
     def __init__(
         self,
         z: float,
-        x: float = 0.,
-        defx: float = 0.,
+        offset: float = 0.,
+        rotation: float = 0.,
```

Comment:
In keeping with the rest of the interface this needs to be degrees. Internally it can remain `Radians` with `@property` setters to handle conversion. (see `STEMSample`, for example).


```suggestion
        rotation: `Degrees` = 0.,
```